migrations/20210816102828_view_ibc_channels.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE view_ibc_channels (
+    id BIGSERIAL,
+    channel_id VARCHAR NOT NULL,
+    port_id VARCHAR NOT NULL,
+    connection_id VARCHAR NOT NULL,
+    counterparty_channel_id VARCHAR NOT NULL,
+    counterparty_port_id VARCHAR NOT NULL,
+    packet_ordering VARCHAR NOT NULL,
+    last_in_packet_sequence BIGINT NOT NULL,
+    last_out_packet_sequence BIGINT NOT NULL,
+    total_transfer_in_count BIGINT NOT NULL,
+    total_transfer_out_count BIGINT NOT NULL,
+    total_transfer_out_success_count BIGINT NOT NULL,
+    total_transfer_out_success_rate FLOAT NOT NULL,
```

Comment:
Here we probably should use NUMERIC because FLOAT has approximate precision.

Reply:
Got it! Let me update this.